project/profiles/capacity4more/behat/features/contact.feature

```diff
@@ -22,4 +22,4 @@ Feature: Contact
     Given I am logged in as user "alfrednobel"
     When I go to user "mariecurie" contact form
     Then I should see "Contact"
-    And I see the wysiwyg
+    And I should see the "#cke_edit-c4m-contact-user-body-value" element
```

Comment:
@ordavidil, are we sure that the wysiwyg exists with this check?

The wysiwyg is added by javascript after the page is loaded. It could happen something before the wysiwyg is loaded preventing it to be added as it happened before.